documentation/how-to/authorization.md

```diff
@@ -0,0 +1,262 @@
+postgresql-k8s      14.7      active       1   postgresql-k8s       14/stable   73  10.152.183.250  no       Primary
+temporal-admin-k8s            active       1   temporal-admin-k8s   stable       4  10.152.183.21   no
+temporal-k8s                  active       1   temporal-k8s         stable       9  10.152.183.191  no
+
+Unit                   Workload   Agent  Address      Ports  Message
+openfga-k8s/0*         active     idle   10.1.232.7
+postgresql-k8s/0*      active     idle   10.1.232.66         Primary
+temporal-admin-k8s/0*  active     idle   10.1.232.71
+temporal-k8s/0*        active     idle   10.1.232.64         auth enabled
+
+Relation provider                 Requirer                       Interface          Type     Message
+openfga-k8s:openfga               temporal-k8s:openfga           openfga            regular
+openfga-k8s:peer                  openfga-k8s:peer               openfga-peer       peer
+postgresql-k8s:database           openfga-k8s:database           postgresql_client  regular
+postgresql-k8s:database           temporal-k8s:db                postgresql_client  regular
+postgresql-k8s:database           temporal-k8s:visibility        postgresql_client  regular
+postgresql-k8s:database-peers     postgresql-k8s:database-peers  postgresql_peers   peer
+postgresql-k8s:restart            postgresql-k8s:restart         rolling_op         peer
+temporal-admin-k8s:admin          temporal-k8s:admin             temporal           regular
+temporal-k8s:peer                 temporal-k8s:peer              temporal           peer
+```
+
+To view your OpenFGA store's information, you can run the command below. Make
+note of the store's information, as this will enable us to connect to it through
+a client to add the necessary tuples in the following section.
+
+```bash
+juju show-unit temporal-k8s/0
+
+# Output:
+temporal-k8s/0:
+  opened-ports: []
+  charm: local:jammy/temporal-k8s-6
+  leader: true
+  life: alive
+  relation-info:
+    ...
+      openfga: '{"store_id": "<store_id>", "token": "<token>",
+        "address": "10.152.183.144", "port": "8080", "scheme": "http", "auth_model_id": "<model_id>"}'
+```
+
+At this point, our Temporal Server is active again with authorization enabled.
+This means that any request made to the Temporal Server will need to be
+[authenticted](./authentication.md) using Google OAuth.
+
+## Create OpenFGA Authorization Rules
+
+Once authorization is enabled, you will not be able to access any namespace in
+Temporal. You must now create the necessary tuples in the OpenFGA store to
+enable users read/write access to different namespaces.
+
+The authorization model used is as follows:
+
+```
+model
+  schema 1.1
+
+type user
+type group
+  relations
+    define member: [user]
+type namespace
+  relations
+    define admin: [group#member]
+    define reader: [group#member] or writer
+    define writer: [group#member]
```

Comment:
`define writer: [group#member] or admin` - it doesn't make sense for an admin to have read-only access.